install_script.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# References
+# http://kvz.io/blog/2013/11/21/bash-best-practices/
+# http://jvns.ca/blog/2017/03/26/bash-quirks/
+
+# exit when a command fails
+set -o errexit
+
+# exit if any pipe commands fail
+set -o pipefail
+
+set -E
+set -o functrace
+function handle_error {
+    local retval=$?
+    local line=${last_lineno:-$1}
+    echo "Failed at $line: $BASH_COMMAND"
+    echo "Trace: " "$@"
+    echo "return code: " "$?"
+    exit $retval
+ }
+trap 'handle_error $LINENO ${BASH_LINENO[@]}' ERR
+
+CONDA_EXE="$(which conda)"
+$CONDA_EXE create -y -n jupyter-vcdat3 -c cdat/label/v81 -c conda-forge nodejs "python>3" vcs jupyterlab pip nb_conda nb_conda_kernels plumbum
+CONDA_BASE=$(conda info --base)
```

Comment:
Not sure what these line do since `conda` is assumed to be in your path from path, sourcing conda.sh wont help much.

Reply:
I had to source the conda.sh as a workaround because the conda executable was not being found in the child process that the script was running in.

Reply:
@William-Hill that's my point, how can `CONDA_EXE="$(which conda)"`  or `CONDA_BASE=$(conda info --base)` work then if it doesn't find the executable?

Reply:
I guess it can find the executable, but not the activate function as mentioned here: https://github.com/conda/conda/issues/7980#issuecomment-441358406

That's what I used as a reference for my workaround.  Sourcing conda.sh makes the activate function available to the process that the script is running in

Reply:
thanks for the explanation, let's merge.